src/carebot-tracker.js

```diff
@@ -304,23 +313,35 @@
             var percent = depthPercent();
             var bucket = percentBucket(percent);
             if (bucket > previousBucket) {
-                var seconds = timer.check().seconds;
-                callback(bucket, seconds);
-            } else {
-                // The user is scrolling back up.
+                callback(bucket, timer.check().seconds);
                 previousBucket = bucket;
```

Comment:
Could you pull this line out of the if statement? The desired behavior is: 
* Report when the user scrolls down the page (working well)
* If the user scrolls back up and starts scrolling down again, the callback is called again for those buckets (requires us to set `previousBucket` no matter if the current bucket is higher or lower)